Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -28,6 +28,7 @@ public class TwineNode : MonoBehaviour
     public string show = "";
     public GameObject[] children;
     [HideInInspector]
+    public string[] childrenNames;
```

Comment:
I renamed "childrenNames" to "linkNames" at one point for accuracy and to avoid confusion.  You have both.